UniAtHome/UniAtHome.WebAPI/Controllers/UserController.cs

```diff
@@ -106,19 +106,5 @@ public async Task<ObjectResult> Revoke()
 
             return Ok(response);
         }
-
-        [Authorize]
-        [HttpGet("check")]
-        public ActionResult<string> CheckAuthorization()
-        {
-            return Ok();
-        }
-
-        [Authorize(Roles = "Admin")]
-        [HttpGet("checkAdmin")]
-        public ActionResult<string> CheckAdmin()
-        {
-            return Ok();
-        }
```

Comment:
And why did you delete these ones too? Testers must have an endpoint to test authorization & authentication.

Reply:
Argee. Fixed.